bundles/org.openhab.binding.teleinfo/src/main/resources/ESH-INF/thing/common-channel-types.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="teleinfo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="isouscType">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>ISOUSC</label>
+		<description>ISOUSC</description>
+		<category>Energy</category>
+		<state pattern="%d A" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="ptecType">
+		<item-type>String</item-type>
+		<label>PTEC</label>
+		<description>PTEC</description>
+		<category>Energy</category>
+		<state pattern="%s" readOnly="true">
+			<options>
+				<option value="TH">TH</option>
```

Comment:
From the user's perspective, it would be better if you translate these values in the tag content...

Reply:
Teleinfo is a french protocol. Here, the "TH" value is an abbreviation for "Toutes les Heures".
For me, it is problematic to use french words in OpenHAB.
IHMO, the user can edit a .map file to translate these values.

Reply:
What about translating them into English?

Reply:
By doing so, we are moving away from the Teleinfo protocol. I don't think it's a good idea.

Reply:
No harm in making a protocol user friendly. Just for clarification, I only suggest to make the UI representation readable `<option value="TH">Hourly</option>`. If you want to stick with it, it should be ok.

Reply:
I think that, for the moment, we must stick strictly to the Teleinfo protocol

Reply:
OK, commit 93073371 reverted by b787e33ba4